modules/apple_support/0.11.0/MODULE.bazel

```diff
@@ -0,0 +1,6 @@
+module(
+    name = "apple_support",
+    version = "0.11.0",
+    compatibility_level = 1,
+)
+bazel_dep(name = "bazel_skylib", version = "1.0.3")
```

Comment:
Interested to see how this version plays out. We can easily bump if needed

Reply:
Isn't this the version that `apple_support` is using as well? https://github.com/bazelbuild/apple_support/blob/c1bcd6a8db95a4df917e8cb1a9c583da8a25b4e4/lib/repositories.bzl#L41

Reply:
yea I'm saying we could easily bump it there if needed

Reply:
Yes!